### PR TITLE
Update the usage messages for gmt defaults, get, and set

### PR DESCRIPTION
**Pull request by PaulWessel**
Follows the rules of #5341.

### CODE REVIEW

File: src/gmtset.c [hunk 1]
```diff
@@ -65,25 +65,29 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct GMTSET_CTRL *C) {	/* Dealloc
...
+	GMT_Usage (API, 3, "s: Use the SI version of defaults.");
+	GMT_Usage (API, 3, "u: Use the US version of defaults.");
+	GMT_Usage (API, 1, "\n-G<defaultsfile>");
+	GMT_Usage (API, -2, "Set name of specific %s file to modify "
+		"[Default looks for file in current directory.  If not found, "
+		"it looks in the home directory, if not found it uses GMT defaults.]", GMT_SETTINGS_FILE);
+	GMT_Usage (API, -2, "Note: Only settings that differ from the GMT SI system defaults are written "
+		"to the file %s in the current directory (under classic mode) "
+		"or in the current session directory (under modern mode).", GMT_SETTINGS_FILE);
+	GMT_Usage (API, 1, "\n-%s", GMT_SHORTHAND_OPTIONS);
```
Comment (meghanrjones) [inline src/gmtset.c:87]: Shorthand options should be -B[JRXYp]<value> rather than -BJRXYp

**Review by meghanrjones — APPROVED** [review]

File: src/gmtset.c (hunk 1 above)
Comment (PaulWessel) [reply src/gmtset.c:87]: Why is that?  GMT_SHORTHAND_OPTIONS cannot be used in that way.

Comment (meghanrjones) [reply src/gmtset.c:87]: Sorry typo. The syntax and online docs have -[BJRXYp]<value>

Comment (PaulWessel) [reply src/gmtset.c:87]: OK, I see.  I am guessing this is a bit lazy and should be -[B|J|R|X|Y|p]<value> ?

Comment (meghanrjones) [reply src/gmtset.c]: > OK, I see. I am guessing this is a bit lazy and should be -[B|J|R|X|Y|p] ?

Yes

**Pull request merged**